README.md

```diff
@@ -14,8 +14,15 @@ This repo currently offers evaluation metrics for two vision tasks:
     - `EceLossEvaluator`: computes the [ECE loss](https://arxiv.org/pdf/1706.04599.pdf), i.e., the expected calibration error, given the model confidence and true labels for a set of data points. 
 - **Object detection**:
     - `MeanAveragePrecisionEvaluatorForSingleIOU`, `evaluators.MeanAveragePrecisionEvaluatorForMultipleIOUs`: computes the mean average precision (mAP), i.e. mean average precision across different classes, under single or multiple [IoU(s)](https://en.wikipedia.org/wiki/Jaccard_index).
-    
+- **Image caption**:
```

Comment:
What are the evaluator classes for those metrics?

Reply:
fixed